### DISCUSSION TITLE
Remember Scroll Position

### DISCUSSION THREAD

**Discussion by JohnLennon3** [ciderapp/Cider#1167] [Feature Request]
### Describe the Bug

When we scroll down on the album or artist page and then click on an album to open it, the app does not remember the scroll position before the click. Hence, when we press "navigate back", the app takes us to the top of the previous page, which is very annoying if we are browsing albums of a particular artist, for example. I've noticed that on the event of the click to open an album, we can see for a very short time the before going to the new page, the current page scrolls to the top. Maybe that is the scroll position that is restored on the navigate back function. 

### Steps to Reproduce

1. Go to the "albums" page to view all albums in the library.
2. Scroll down a bit 
3. Open any album by clicking on it
4. Press the "navigate back" arrow, to return to the "albums" page
5. The app restores the previous page at the top, no matter what the previous scroll position was
6. It does the same thing on the "artists" page

### Environment Information

- OS: Windows 10
- App Version and/or Commit ID: 1.5

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by coredev-uk** [comment]
This is more of a feature than a bug for that app. Remembering states on exact pages could be something made further down the line. Moving to dicussion.
